modules/ROOT/pages/traversal-framework/traversal-framework-example.adoc

```diff
@@ -119,7 +119,7 @@ This will give the following output:
 (0)-[KNOWS,0]->(2)<-[KNOWS,2]-(3)<-[KNOWS,3]-(4)<-[KNOWS,4]-(1)
 ----
 
-To see other useful evaluators, check the link:{org-neo4j-graphdb-traversal-Evaluators}[`org.neo4j.graphdb.traversal.Evaluators`^] Java API or simply implement the link:{org-neo4j-graphdb-traversal-Evaluator}[`org.neo4j.graphdb.traversal.Evaluator`^] interface yourself.
+To see other useful evaluators, check the xref:/docs/java-reference/{neo4j-version}/traversal-framework/traversal-framework-java-api/#traversal-java-api-evaluator[`org.neo4j.graphdb.traversal.Evaluators`^] Java API or simply implement the  xref:/docs/java-reference/{neo4j-version}/traversal-framework/traversal-framework-java-api/#traversal-java-api-evaluator[`org.neo4j.graphdb.traversal.Evaluator`^] interface yourself.
```

Comment:
```suggestion
To see other useful evaluators, see xref:traversal-framework/traversal-framework-java-api.adoc#traversal-java-api-evaluator[Using Evaluators].
// For details of the interface, see the  xref:/docs/java-reference/{neo4j-version}/traversal-framework/traversal-framework-java-api/#traversal-java-api-evaluator[`org.neo4j.graphdb.traversal.Evaluator`^] javadocs.
```

A couple of notes on this suggestion:
- The first link corrects the format for an internal link in the manual
- The second link is to a page in the javadocs that will exist, but only after the next API update is released and published. Maybe for now we can comment it out and add it back in after the target page is published.

Reply:
Great! Thanks Neil.